bin/lint-migrations-file/deps.edn

```diff
@@ -1,8 +1,7 @@
 {:paths ["src"]
 
  :deps
```

Comment:
Based on your comments, can we also upgrade snakeyaml to 2.0? If so, would you mind doing it in this PR?